phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.css

```diff
@@ -644,6 +644,11 @@
 /************************ icons ************************/
 /*******************************************************/
 
+/************************ util *************************/
+.refreshIcon {
+    -fx-graphic: url('/org/phoenicis/javafx/themes/breezeDark/icons/mainwindow/refresh.png');
```

Comment:
Why doesn't it belong in the scope of this PR? It's a part of the refactoring. The refresh icon was used before and it's used afterwards. It's just that I've moved the icon graphic definition to the css files.